torchvision/__init__.py

```diff
@@ -3,7 +3,7 @@
 from torchvision import transforms
 from torchvision import utils
 
-__version__ = '0.2.0'
+__version__ = '0.2.0.bkj'
```

Comment:
Hum, I'm not sure we should be manually modifying the `__version__`, ideally we should use the commit hashes, but that's for another story. Can you maybe revert this?